packages/server/src/shelf/shelf.controller.ts

```diff
@@ -106,10 +106,24 @@ export default class ShelfController extends BaseController {
       this.serverApi.shelfGetLastActions,
       authMiddleware,
       this.getLastActionsHanlder
-    )
+    );
+
+    this.router.post(
```

Comment:
Following our conversation today maybe we could start using `GET`s to get things.

Reply:
Check sluck